lib/ansible/plugins/doc_fragments/default_callback.py

```diff
@@ -15,6 +15,7 @@ class ModuleDocFragment(object):
         default: yes
         env:
           - name: DISPLAY_SKIPPED_HOSTS
```

Comment:
Have you verified that the deprecation warnings display without also adding the deprecation info to the config snippet for the plugin?

Reply:
I've verified `ansible-config dump --only-changed` issues the warning.

Reply:
Also:
```
ansible localhost -m debug -a msg=test
[DEPRECATION WARNING]: DISPLAY_SKIPPED_HOSTS option, environment variables without "ANSIBLE_" prefix are deprecated , use the "ANSIBLE_DISPLAY_SKIPPED_HOSTS" environment variable instead. This feature will be removed in version 2.12.
Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
localhost | SUCCESS => {
    "msg": "test"
}
```